### ISSUE TITLE
Implement an incremental based armcontroller.

### ISSUE THREAD

**Issue by OPstriker** [GoldenDragons772/CenterStage#53]
Currently, we have pre-set heights for the arms, which works well for most use cases except when it breaks. The other proposed option is to use a manual control-based arm that uses triggers to make the arm go up and down manually. With this method, the driver retains control over the arm and allows them to adjust the arm or reset to the starting position and fix the slide if the starting position gets messed up. The arm would essentially go up or down when the user presses the desired trigger and when the user releases the trigger the arm should stop and maintain the current position based on the encoder position/PID output.

TLDR;
We need something that allows the driver to fine-adjust the arm position and maintain it once they let go of the controller.
